.semaphore/semaphore.yml

```diff
@@ -0,0 +1,58 @@
+version: v1.0
+name: "Checkstyle CI pipeline on Semaphore"
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+auto_cancel:
+  running:
+    when: "branch != 'master'"
+blocks:
+  - name: "Linux openjdk 11 build"
+    task:
+      prologue:
+        commands:
+          - checkout
+          - cache restore m2
+          - eval "export M2_CACHE_SIZE=$(du -s $HOME/.m2 | cut -f1)"
+          - sudo apt-get update
+          - sudo apt-get install -y ant groovy xsltproc xmlstarlet
+          - sem-version java 11
+      epilogue:
+        commands:
+          - |
+            if [[ ${M2_CACHE_SIZE} -ne $(du -s $HOME/.m2 | cut -f1) ]]
+            then
+              cache delete m2
+              cache store m2 $HOME/.m2
+            fi
+      jobs:
+        - name: Ensure that all Sevntu check are used
+          commands:
+            - .ci/semaphore.sh all-sevntu-checks
+
+        - name: No error test on pmd
+          commands:
+            - .ci/semaphore.sh no-error-pmd
+
+        #  Disabled due to instability
+        #  - name: No violation testing on josm
+        #    commands:
+        #      - .ci/semaphore.sh no-violation-test-josm
+
+        - name: No exception test
+          matrix:
+            - env_var: PROJECT
+              values:
+                - checkstyle-sevntu-checkstyle
+                - guava
+                - guava-with-google-checks
+                - guava-with-sun-checks
+                - hibernate
+                - spotbugs
+                - spring-framework
+                - hbase
+                - pmd-elasticsearch-lombok-ast
+                - alot-of-project1
+          commands:
+            - .ci/semaphore.sh no-exception-test-$PROJECT
```

Comment:
As we started moving execution, 
It makes sense to do
./no-exception-test.sh alot-of-project1

And any CI can execute such command, even user ecute validation command.
And we can easily move commands between CI without modification of she'll files.

Reply:
done